doc/users/interactive.rst

```diff
@@ -57,64 +58,58 @@ IPython integration
 ===================
 
 We recommend using IPython for an interactive shell.  In addition to
-all of its features (improved tab-completion, magics,
-multiline editing, etc), it also ensures that the GUI toolkit event
-loop is properly integrated with the command line (see
-:ref:`cp_integration`).  To configure the integration and enable
-:ref:`interactive mode <controlling-interactive>` use the
-``%matplotlib`` magic
+all of its features (improved tab-completion, magics, multiline editing, etc), 
+it also ensures that the GUI toolkit event loop is properly integrated 
+with the command line (see :ref:`cp_integration`). 
+
+In this example, we are modifying a figure via an IPython prompt. 
+The figure is displayed in a Qt5Agg GUI Window. To configure the integration 
+and enable :ref:`interactive mode <controlling-interactive>` use the
+``%matplotlib`` magic:
```

Comment:
'modifying' implies it already exists, but the example is creating from scratch and then modifying. Also, the verb tense changes later, so switch that here as well.

```suggestion
In this example, we create and modify a figure via an IPython prompt. 
The figure displays in a Qt5Agg GUI window. To configure the integration 
and enable :ref:`interactive mode <controlling-interactive>` use the
``%matplotlib`` magic:
```